docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -211,6 +211,9 @@ a few caveats:
   - Enable using Apache Arrow serialization when reading data from BigQuery.
     Read this [section](bigquery-arrow-serialization-support) before using this feature.
   - `true`
+* - `bigquery.max-parallelism`
+  - The max number of partitions to split the data into.
```

Comment:
this is per-table scan right? Not per query, correct?

Reply:
Correct, it's per-table.

Reply:
Nit: why this description is different from the session and config properties?

Reply:
Let's add a more throrough documentation for this property.
Ideally one should know when it is the case to employ it